test.php

```diff
@@ -177,6 +177,35 @@
 $data['nothing'] = $ck->query('select array()');
 
 
+$data['drop table']   = $ck->query('DROP TABLE IF EXISTS t5');
+$table                = [
+    'CREATE TABLE t5 (',
+    '`id` UInt32,',
+    '`f1` UInt16,',
+    '`f2` SimpleAggregateFunction(max, DateTime64)',
+    ') ENGINE = AggregatingMergeTree() ORDER BY id, f1 SETTINGS index_granularity = 8192'
```

Comment:
There seems to be a mistake here
`) ENGINE = AggregatingMergeTree() ORDER BY (id, f1) SETTINGS index_granularity = 8192`
If there is no line break, you need to add (). `(id, f1)`

Reply:
> There seems to be a mistake here
> `) ENGINE = AggregatingMergeTree() ORDER BY (id, f1) SETTINGS index_granularity = 8192`
> you need to add (). `(id, f1)`

Sorry it's my mistake.
One minute fix.